CHANGELOG.md

```diff
@@ -54,14 +54,22 @@ This version adds a new flag `-mysql_max_idle_conns` to specify the number of
 idle database connections in the pool. Defaults to -1 which uses the Go default.
 Go default is currently set at 2 but could change in future releases.
 
+### Server Verification
```

Comment:
That comment is a bit generic. Can we add Hash or something?

Reply:
I've changed this heading to "Server verification of leaf hashes".

Reply:
Maybe 'validate' is better as it doesn't raise the spectre of crypto like verification does?

Reply:
Should we also change the "client verification" section just below, as that appears to have a similar issue?

Reply:
Might as well, it was also a very generic title.

Reply:
Done.